src/main/java/no/unit/nva/model/contexttypes/Book.java

```diff
@@ -15,7 +17,7 @@
 import org.apache.commons.validator.routines.ISBNValidator;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public class Book implements BasicContext {
+public class Book implements BasicContext, LinkedContext {
```

Comment:
In an example of a BookAnthology and a Chapter, Would the Chapter have a linkedContext pointing to the BookAnthology ? If it is not this case but the opposite, we should have a collection of linkedContexts

Reply:
Added an sample with a Book in a series, but there will/must be upcoming changes in the model regardin handling of series

Reply:
The linkedContext is  always X isPartOf Y, not Y hasPart X

Reply:
Moreover, if we want to represent these things in terms of the thing that has the hasPart relationship, I consider this to be an index task as there is no guarantee that that we know the parts (example, the editor of an anthology, where all the contributors are not at Norwegian institutions)